{
  "affected": [
    {
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/discordbotstatus/MAL-2025-920.json"
      },
      "package": {
        "ecosystem": "PyPI",
        "name": "discordbotstatus",
        "purl": "pkg:pypi/discordbotstatus"
      },
      "versions": [
        "0.6.7"
      ]
    }
  ],
  "credits": [
    {
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "name": "ReversingLabs",
      "type": "FINDER"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2025-00459",
        "import_time": "2025-02-03T18:38:05.978064485Z",
        "modified_time": "2025-02-03T17:07:18Z",
        "sha256": "86f8cdb3ebd03eb15ffca651714087164178302fd28712d8f93db9afc1efa534",
        "source": "reversing-labs",
        "versions": [
          "0.6.7"
        ]
      }
    ]
  },
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "id": "MAL-2025-920",
  "modified": "2025-02-03T17:07:18Z",
  "published": "2025-02-03T17:07:18Z",
  "schema_version": "1.6.0",
  "summary": "Malicious code in discordbotstatus (PyPI)"
}